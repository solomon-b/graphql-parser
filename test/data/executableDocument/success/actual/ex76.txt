[ Left
    ( Left
        ( OperationDefinition
            { opType = Query
            , opName = Nothing
            , opVariables = []
            , opDirectives = []
            , opSelectionSet = Left
                ( Left
                    ( Field
                        { fieldAlias = Nothing
                        , fieldName = "firstSearchResult"
                        , fieldArguments = fromList []
                        , fieldDirectives = []
                        , fieldSelectionSet = Just
                            ( Right
                                ( InlineFragment
                                    { ifTypeCondition = Just
                                        ( TypeCondition
                                            { unTypeCondition = "Person" }
                                        )
                                    , ifDirectives = []
                                    , ifSelectionSet = Left
                                        ( Left
                                            ( Field
                                                { fieldAlias = Nothing
                                                , fieldName = "name"
                                                , fieldArguments = fromList []
                                                , fieldDirectives = []
                                                , fieldSelectionSet = Nothing
                                                }
                                            )
                                        ) :| []
                                    }
                                ) :|
                                [ Right
                                    ( InlineFragment
                                        { ifTypeCondition = Just
                                            ( TypeCondition
                                                { unTypeCondition = "Photo" }
                                            )
                                        , ifDirectives = []
                                        , ifSelectionSet = Left
                                            ( Left
                                                ( Field
                                                    { fieldAlias = Nothing
                                                    , fieldName = "height"
                                                    , fieldArguments = fromList []
                                                    , fieldDirectives = []
                                                    , fieldSelectionSet = Nothing
                                                    }
                                                )
                                            ) :| []
                                        }
                                    )
                                ]
                            )
                        }
                    )
                ) :| []
            }
        )
    )
]