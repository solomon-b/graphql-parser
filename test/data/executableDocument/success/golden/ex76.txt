[ Left
    ( Left
        ( OperationDefinition
            { opType = Query
            , opName = Nothing
            , opVariables = []
            , opDirectives = []
            , opSelectionSet = Left
                ( Left
                    ( Field
                        { fieldAlias = Nothing
                        , fieldName = "firstSearchResult"
                        , fieldArguments = fromList []
                        , fieldDirectives = []
                        , fieldSelectionSet = Just
                            ( Right
                                ( InlineFragment
                                    { ifTypeCondition = Just "Person"
                                    , ifDirectives = []
                                    , ifSelectionSet = Left
                                        ( Left
                                            ( Field
                                                { fieldAlias = Nothing
                                                , fieldName = "name"
                                                , fieldArguments = fromList []
                                                , fieldDirectives = []
                                                , fieldSelectionSet = Nothing
                                                }
                                            )
                                        ) :| []
                                    }
                                ) :|
                                [ Right
                                    ( InlineFragment
                                        { ifTypeCondition = Just "Photo"
                                        , ifDirectives = []
                                        , ifSelectionSet = Left
                                            ( Left
                                                ( Field
                                                    { fieldAlias = Nothing
                                                    , fieldName = "height"
                                                    , fieldArguments = fromList []
                                                    , fieldDirectives = []
                                                    , fieldSelectionSet = Nothing
                                                    }
                                                )
                                            ) :| []
                                        }
                                    )
                                ]
                            )
                        }
                    )
                ) :| []
            }
        )
    )
]